modules/javafx.controls/src/test/java/test/javafx/scene/control/TooltipTest.java

```diff
@@ -706,4 +706,8 @@ private void assertTooltipHiddenAfter(Rectangle rect, int millis) {
         assertFalse(toolTip.isShowing());
     }
 
+    private String toBase64(String css) {
```

Comment:
suggestion:
```
private static String encode(String css) {
  return "data:base64," + Base64.getUrlEncoder().encodeToString(css.getBytes(StandardCharsets.UTF_8));
}
```

Reply:
I agree that it could be static, although I prefer the name `toBase64`.

Reply:
my comment was to avoid replicating "data:base64,", and with that, it is not base64 anymore, but base-64-encoded url.

the best place for this method is probably in some utility class so it can be reused.

Reply:
created https://bugs.openjdk.org/browse/JDK-8336334